src/main/resources/application.properties

```diff
@@ -16,4 +40,16 @@ spring.datasource.hikari.maximum-pool-size=10
 spring.jpa.properties.hibernate.show_sql=true
 spring.jpa.properties.hibernate.format_sql=true
 spring.jpa.hibernate.ddl-auto=update
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
\ No newline at end of file
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+
+# JWT secret key (you should use a strong, random string)
+app.jwt.secret=u4xvJDMA2ukH0P3cN8oRbgu4xvJDMA2ukH0P3cN8oRbgu4xvJDMA2ukH0P3cN8oRbgu4xvJDMA2ukH0P3cN8oRbgu4xvJDMA2ukH0P3cN8oRbg
```

Comment:
secrets shouldn't be here. should be inferred from an env

Reply:
yeah i'll do that on merge

Reply:
How can you do this on merge if approval is required on merge?

Reply:
i mean ill create another merge, later.

Reply:
for the secrets stuff

Reply:
working on it

Reply:
done